app/docs/components/card/card.mdx

```diff
@@ -67,6 +67,27 @@ Add an image to the card by using the `imgSrc` prop and set the `imgAlt` prop to
   </Card>
 </CodePreview>
 
+## Card with custom image render function
+
+Specify your own render function for the image component for the card by using the `renderImage` prop. This is especially useful when
+using the component with NextJS or Gatsby.
+
+<CodePreview
+  importFlowbiteReact="Card"
+  importExternal="import Image from 'next/image'"
+  title="Card with decorative image"
+  className="max-w-sm"
+>
+  <Card renderImage={() => <Image width={500} height={500} src="/images/blog/image-1.jpg" />}>
```

Comment:
Unfortunately this does not work as expected but there is little I can do about this. The `CodePreview` does not properly support external dependencies.
![Uploading Screenshot 2023-07-06 at 22.50.47.png…]()

Reply:
@tulup-conner @zoltanszogyenyi any idea?

Reply:
Great stuff. Looks better now.
![Screenshot 2023-07-12 at 08 15 18](https://github.com/themesberg/flowbite-react/assets/1150767/3695d109-9cb9-4b49-9c9e-2719fd0dedeb)